WooCommerce/Classes/ViewRelated/Notifications/Cells/NoteDetailsHeaderPlainTableViewCell.swift

```diff
@@ -48,16 +48,38 @@ final class NoteDetailsHeaderPlainTableViewCell: UITableViewCell {
 
     override func awakeFromNib() {
         super.awakeFromNib()
+        configureBackground()
+        configureImages()
+        configureLabels()
+    }
+}
 
+// MARK: - Private Methods
+//
+private extension NoteDetailsHeaderPlainTableViewCell {
+
+    /// Setup: Cell background
+    ///
+    func configureBackground() {
+        applyDefaultBackgroundStyle()
+    }
+    
+    /// Setup: Images
+    ///
+    func configureImages() {
         imageView?.tintColor = StyleManager.wooGreyTextMin
         accessoryImageView.tintColor = StyleManager.wooCommerceBrandColor
         accessoryView = accessoryImageView
+    }
+    
+    /// Setup: Labels
+    ///
+    func configureLabels() {
         textLabel?.font = UIFont.body
         textLabel?.textColor = StyleManager.defaultTextColor
     }
 }
 
-
```

Comment:
Please restore this line? Our styles include a double-space before using a pragma mark.